File: web/scripts/adm/dynamic.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */


$(document).ready(function (){
    $('input[name=Add]').click(function(){
        //alert("hurrayy");
        var str='<tr><td>Paper Id</td>';
            str +='<td><input type="text" name="paperid" value="" /></td>';
            str +='<td>Paper Name</td>';
            str += '<td><input type="text" name="papername" value="" /></td></tr>';
                        
        $('#papers > tbody:last').append(str);
    })
});

File: src/java/shg/bean/Dynamic.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package shg.bean;

import shg.valid.Validator;

/**
 *
 * @author B Mukhim
 */
public class Dynamic {
    private String [] paperid;
    private String [] papername;
    

    /**
     * @return the paperid
     */
    public String[] getPaperid() {
        return paperid;
    }

    /**
     * @param paperid the paperid to set
     */
    public void setPaperid(String[] paperid) {
        this.paperid = paperid;
    }

    /**
     * @return the papername
     */
    public String[] getPapername() {
        return papername;
    }

    /**
     * @param papername the papername to set
     */
    public void setPapername(String[] papername) {
        this.papername = papername;
    }
    
    public boolean isPaperidValid(){
        int len=getPaperid().length;
        String id[]=getPaperid();
        for(int i=0; i< len; i++){
            if(Validator.isNullValue((id[i]))){
                return false;
            }
        }
        return true;
    }
}
